tests/integration/app.test.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const { setupStrapi } = require("./strapi");
+const request = require("co-supertest");
+
+/** this code is called once before any test is called */
+before((done) => {
+  done()
+  return setupStrapi();
+});
+
+it("strapi is defined", async (done) => {
+  console.log("strapi is defined");
+  done();
+});
+
+require("./controllers/activity.test.js");
```

Comment:
why mentioned individual files here controller folder???
just include the whole folder instead so all the files will be internally added automatically.
refer https://github.com/qunabu/strapi-unit-test-example/blob/729c877cedbf8fecd414876c57fe7f790c041ee0/tests/app.test.js

Reply:
they have called directly the folder because inside that folder they only have one file called index.js so it run's properly but in our case it throws error stating " Cannot find module './controllers' "